packages/react-devtools-shared/src/devtools/views/Profiler/RecordToggle.js

```diff
@@ -30,13 +30,17 @@ export default function RecordToggle({disabled}: Props): React.Node {
     className = styles.ActiveRecordToggle;
   }
 
+  const isMac = navigator.platform.toUpperCase().indexOf('MAC') >= 0;
```

Comment:
We check the same thing differently in another part of the library. See https://github.com/facebook/react/blob/fb2177c153353621c5e343b0386993e5084f641e/packages/react-devtools/app.js#L38